api-docs/cloud-dedicated/ref.yml

```diff
@@ -1863,13 +1917,30 @@ paths:
         required: true
       responses:
         "204":
-          description: Write data is correctly formatted and accepted for writing to the database.
+          description: |
+            Success. All points in the batch are written and queryable.
         "400":
           content:
             application/json:
+              examples:
+                partialWriteErrorWithRejectedPoints:
```

Comment:
Same suggested changes:
* have 2 examples (rejected points, vs namespace)
* `partialWriteErrorWithRejectedPoints` => `writeErrorWithRejectedPoints`, with associated wording changes.

Reply:
Not configurable; no change required.